src/commands.rs

```diff
@@ -1558,8 +1558,9 @@ fn log_template(settings: &UserSettings) -> String {
     };
     let default_template = format!(
         r#"
-            change_id.short()
-            if(divergent, label("divergent", " divergent"))
+            if(divergent,
+              label("divergent", change_id.short() "??"),
+              change_id.short())
```

Comment:
Might be worth mentioning in the changelog, in case it helps explain to current users where the "divergent" label in the log output went.

Reply:
Will do.